### PR TITLE
[BUGFIX] - Stimpack recipes for synthetics and humans

### PR DESCRIPTION
**Pull request by Rayfod**
## About The Pull Request
A few stimpack recipes were added since the super stimpack needed synthetic blood while the normal stimpack required blood.
Now you can make both with synthetic blood and normal blood.

Code dev request: removed the code instead of *\  /* as requested by a head code to clean up the code.

### CODE REVIEW

**Comment by SigholtStarsong** [comment]
Would you mind terribly following the format provided on your PR's so that it's a little easier to understand exactly what you're doing?

**Comment by Rayfod** [comment]
Tested: Worked

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 1]
```diff
@@ -374,79 +374,36 @@
...
 /datum/chemical_reaction/superstimpak
 	name = "Super Stimpak Fluid"
 	id = /datum/reagent/medicine/super_stimpak
-	results = list(/datum/reagent/medicine/super_stimpak = 5)
+	results = list(/datum/reagent/medicine/super_stimpak = 2)
+	required_reagents = list(/datum/reagent/blood = 1, /datum/reagent/medicine/stimpak = 1, /datum/reagent/consumable/mutjuice = 1) //1 mutfruit at 50 potency yields 6 mutfruit juice
+
+/datum/chemical_reaction/superstimpak2
+	name = "Synthetic Super Stimpak Fluid"
+	id = /datum/reagent/medicine/super_stimpak
```
Comment (out-of-phaze) [inline code/modules/reagents/chemistry/recipes/medicine.dm:398]: This is a duplicate id; IDs for reactions must be unique. Use a string, i.e. `"superstimpak2"` or `"synthetic_super_stimpak"`.

**Review by out-of-phaze — DISMISSED** [review]

File: code/modules/reagents/chemistry/recipes/medicine.dm (hunk 1 above)
Comment (Rayfod) [reply code/modules/reagents/chemistry/recipes/medicine.dm]: This should probably fix. Tested and worked.

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 2]
```diff
@@ -374,79 +374,32 @@
...
-	ThermicConstant		= -6 //Temperature change per 1u produced
-	HIonRelease 		= -0.1 //pH change per 1u reaction
-	RateUpLim 			= 12 //Optimal/max rate possible if all conditions are perfect
-	FermiChem 			= TRUE//If the chemical uses the Fermichem reaction mechanics
-	FermiExplode 		= FALSE //If the chemical explodes in a special way
-*/
+
+/datum/chemical_reaction/stimpak/synthetic
+	id = "stimpak_synthetic"
+	required_reagents = list(/datum/reagent/blood/synthetics = 1, /datum/reagent/medicine/spaceacillin = 1)
```
Comment (ProbablyCarl) [inline code/modules/reagents/chemistry/recipes/medicine.dm:386]: No results / name?

Comment (Rayfod) [reply code/modules/reagents/chemistry/recipes/medicine.dm:386]: It's inherited by the /stimpak
Kind of like 
```
/datum/chemical_reaction/synthflesh
	name = "Synthflesh"
	id = /datum/reagent/medicine/synthflesh
	results = list(/datum/reagent/medicine/synthflesh = 3)
	required_reagents = list(/datum/reagent/blood = 1, /datum/reagent/carbon = 1, /datum/reagent/medicine/styptic_powder = 1)
```
	
```
/datum/chemical_reaction/synthflesh/synthblood
	id = "synthflesh_2"
required_reagents = list(/datum/reagent/blood/synthetics = 1, /datum/reagent/carbon = 1, /datum/reagent/medicine/styptic_powder = 1)
```

Comment (Rayfod) [reply code/modules/reagents/chemistry/recipes/medicine.dm:386]: It makes the code look a little cleaner, which is why I choose to do it.
And it worked fine in the test for the use of synth blood to stimpak

**Comment by ProbablyCarl** [comment]
Test merged briefly.

**Review by ProbablyCarl — APPROVED** [review]

**Pull request merged**